doc/api-details.md

```diff
@@ -176,6 +176,23 @@ $ curl -X 'PUT' \
 {"_id":"61bda8f2eb1a63d2b7152418","kind":"node","name":"checkout-test","revision":{"tree":"mainline","url":"https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git","branch":"master","commit":"2a987e65025e2b79c6d453b78cb5985ac6e5eb26","describe":"v5.16-rc4-31-g2a987e65025e"},"parent":null,"status":"pending","result":null, "created":"2022-02-02T11:23:03.157648", "updated":"2022-02-02T12:23:03.157648"}
 ```
 
+### Getting Nodes count
+
+To get a count of all the nodes, use GET request to `/count` endpoint.
+
+```
+$ curl http://localhost:8001/count
+4
+```
+
+To get count of nodes matching provided attributes, use `/count` endpoint with query parameters.
+```
+$ curl http://localhost:8001/count?name=checkout
+3
+$ curl http://localhost:8001/count?revision.branch=staging-mainline
+1
```

Comment:
I think it'd be good to put a bit more information here:
- Where these parameters come from/What parameters can be used ?
- Is it possible to use more than one parameter in a query and how it'll be interpreted?

Reply:
Updated the details. Please check.